### ISSUE TITLE
`spaces.Discrete.sample` never returns the top value

### ISSUE THREAD

**Issue by epignatelli** [epignatelli/navix#210]
_Filed by an AI agent on behalf of @epignatelli, from findings during the #167 documentation pass (see PR #209)._

`Discrete.create(n_elements)` stores `maximum = n_elements - 1`, and `Discrete.sample` calls:

```python
jax.random.randint(key, self.shape, self.minimum, self.maximum)
```

`jax.random.randint`'s `maxval` is **exclusive**, so the draw is over `[0, n_elements - 2]` — the value `n_elements - 1` can never be sampled.

Impact: `env.action_space.sample()` never returns the last action in `action_set`; any code sampling a `Discrete` observation/label space is biased away from the top class.

Fix: pass `self.maximum + 1` as `maxval` (or store `maximum = n_elements` and adjust `.n`). Add a test that `set(Discrete.create(k).sample(...))` covers `0..k-1` over enough draws.
